### ISSUE TITLE
[Screen Reader - Left Navigation]: Screen reader is not reading State and position details of menu items under left navigation bar.

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#7505]
User Experience:
If screen reader is not reading the state and position details for menu items under left navigation bar then the end user who depends upon screen reader will not be able to know whether the current element is selected or not and might not know how much further he/she needs to navigate under this section.

Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

Repro Steps:
1. Open URL :https://preview.flow.microsoft.com and sign in using valid credentials.

2. Navigate to Hamburger button on the left and select it.
3. Left Navigation pane appears, navigate to the All menu item under this section.​

4. Turn on Narrator and navigate on to the "Home" menu item.​
5. Observe the screen reader behavior.

Actual Result:
Screen reader is only reading "Home Menu item" as the details of element under Left navigation bar.

Expected Result:
Screen reader should read "Home Menu item selected 1 of 8", "Approvals Menu item non-selected 2 of 8" etc. as the details of element under Left navigation bar.

Note:
1. Same issue exists for "Data Expand/Collapsed" menu item which is appearing under Left navigation bar.
2. Same issue exists with Chrome +Jaws.

![leftnavigationbar](https://user-images.githubusercontent.com/30456910/50591936-0c9e8f00-0eb8-11e9-81d1-e4cac5490ee8.png)

**Comment by cliffkoh** [comment]
Hi @sonimishra, I am unable to tell which Fabric components are involved here. Can you please create a repro in a codepen?

**Comment by cliffkoh** [comment]
Hi @sonimishra, apologies, I realize this is for the Fabric Nav component. The issues here will have to be addressed as part of a complete Nav rewrite.

**Comment by cliffkoh** [comment]
The above comment is if the fixes for the issues are pursued by the Fabric Tactical team. PRs are always welcome if there is a pressing need to resolve the issues above.
